guardian/tagmanager
# Bump dependency versions to sort vulnerabilities

## What does this change?

This PR bumps a few packages to sort some vulnerabilities reported by Snyk.

Some Webpack-associated dependencies had to be bumped alongside Webpack.

We'll need to remove Scribe to sort the final high-priority vulnerability.

## How to test

Run the project locally according to the instructions in the readme. Does it build as expected? (I have tested locally and it worked for me.)
